docs/installation/introduction.rst

```diff
@@ -258,26 +261,25 @@ To achieve this, use the following commands after deployment:
 
 Networks
 """"""""
+
 A MetalK8s cluster requires a physical network for both the control plane and
 the workload plane Nodes. Although these may be the same network, the
 distinction will still be made in further references to these networks, and
-when referring to a Node IP address. Each Node in the cluster **must** belong
+when referring to a Node IP address. Each Node in the cluster *must* belong
```

Comment:
Do we have guidelines for this kind of formatting as well? I honestly prefer bold than italic to emphasize verbs.

Reply:
I'll revisit our style guide on this. The rule I follow is that emphases are italic, not bold (like the <em> tag does). We reserve boldface for GUI elements.